oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/ElasticsearchIndexDefinition.java

```diff
@@ -85,7 +86,7 @@ public String getRemoteIndexName() {
 
     private String setupIndexName() {
         // TODO: implement advanced remote index name strategy that takes into account multiple tenants and re-index process
-        return getESSafeIndexName(getIndexPath() + "-" + getReindexCount());
+        return getESSafeIndexName(indexPrefix + getIndexPath() + "-" + getReindexCount());
```

Comment:
should we add a `.` to separate `indexPrefix` from `indexPath`?

Reply:
`{getESSafeName` converts `/` to `_` and indexPath starts with `/` so these two are already separated. Eg if indexPrefix is `15Apr` and index name is `eIndexOne` then remote index name is `apr15_eindexone-1`

Reply:
it's OK then

Reply:
@averma21 I thought more about this. I think we should add a `.` anyway because `indexPath` starting with `/` is not always true.
Looking at the benchmarks, the final index name will be `benchmarkelasticglobal$ms`. We always need a fixed delimiter for the role permissions. Probably we will need additional validation on the prefix (eg: it cannot contains dots) but we can improve this in a separate PR.